cranelift-codegen/meta/src/shared/instructions.rs

```diff
@@ -1318,6 +1353,26 @@ pub fn define(
         .other_side_effects(true),
     );
 
+    let N = &operand_doc(
+        "args",
+        variable_args,
+        "Variable number of args for Stackmap",
+    );
+
+    ig.push(
+        Inst::new(
+            "savepoint",
```

Comment:
I am slightly wondering if there is a typo here and if this name was supposed to be "safepoint", since it's usually more used in the compiler literature for such an instruction.

Reply:
Yes, this was a very consistent misspelling. Fixed in the latest commit. :)